<launch>
	<!-- 
	You can use the ROS image viewer to look at the live video, for example:
	 
	rosrun image_view image_view image:=camera/image_raw
	rosrun image_view image_view image:=camera/image_rect
	rosrun image_view image_view image:=camera/image_foreground
	rosrun image_view image_view image:=camera/image_processed
	-->
	  
	<param name="frameid_contours" type="string" value="ImageRect"/>

	<param name="tooloffset/kP"      type="double" value="1.0"/>
	<param name="tooloffset/kI"      type="double" value="0.0"/>
	<param name="tooloffset/kD"      type="double" value="0.0"/>

	<param name="tracking/speedThresholdForTravel" type="double" value="5.0"/>
	<param name="tracking/dtVelocity"              type="double" value="0.2"/>

	<node pkg="tracking" type="ContourGenerator.py" name="ContourGenerator" />
	<node pkg="tracking" type="ContourIdentifier.py" name="ContourIdentifier" />
	
	<!-- Causes ContourIdentifier to hang when run w/ no Stage:  -->
	<!-- <node pkg="tracking" type="PublishFlyView.py" name="PublishFlyView" /> --> 
</launch>
